src/fireblocks-sdk.ts

```diff
@@ -1820,4 +1848,78 @@ export class FireblocksSDK {
     private getCommaSeparatedList(items: Array<string>): string | undefined {
         return items ? items.join(",") : undefined;
     }
+
+    /**
+     * Get list of jobs for current tenant
+     * @param fromTime  beggining of time range in MS since 1970
+     * @param toTime    ending of time range in MS since 1970
+     */
+    public getJobs(fromTime: number, toTime: number): Promise<Job[]> {
```

Comment:
getJobs is way to general. Also, please separate this endpoint from the next one with a better name (`getApplePickingJobsBetweenTimestamps`)